qase-tms/qase-codeception
# Logging debug option support

How to test:
```
git clone https://github.com/in4s/qase-codeception.git
cd qase-codeception/example
git checkout logging_debug_option_support
composer install --prefer-dist
```
Then make changes to the config file: `examples/.env.test`
And run tests:
`php vendor/bin/codecept run `

I guess it needs to be tested only three things:
1. Config with `QASE_LOGGING=1`
2. Config with `QASE_LOGGING=0`
3. Config without `QASE_LOGGING=1` line


Variants 1 and 3 should show the reporter log in console, as usual. 2nd variant should not show the reporter log. But all three variants should send data to Qase.
